driver/lib/commands/assertions.ts

```diff
@@ -0,0 +1,85 @@
+import { FlutterDriver } from '../driver';
+import { waitFor, waitForAbsent, waitForTappable } from './execute/wait';
+import { byValueKey, byText, byTooltip, serializeFinder } from './finder';
```

Comment:
This import doesn't work. You would need to add new npm dependency and import from that.
https://www.npmjs.com/package/appium-flutter-finder